pom.xml

```diff
@@ -126,6 +126,9 @@
         <!-- GitHub -->
         <github.namespace>wildfly-extras</github.namespace>
         <github.project>wildfly-camel</github.project>
+
+        <!-- Common Maven properties -->
+        <project.build.sourceEncoding>utf-8</project.build.sourceEncoding>
```

Comment:
Nitpicking a bit here, but can we go with uppercase `UTF-8` for consistency.

Reply:
"for consistency" with what?

Reply:
Consistency with:

* How it's written in the rest of the project
* The example in official maven docs https://maven.apache.org/general.html
* Every other [maven project](https://github.com/search?l=Maven+POM&p=1&q=sourceEncoding&type=Code&utf8=%E2%9C%93) on the planet :wink: